code/modules/multiz/movement.dm

```diff
@@ -89,6 +90,13 @@
 		return
 	. = ..()
 	if(.)
+		// Chance to fail while trying to climb down a lattice structure or ladder
+		var/turf/T = GetAbove(src)
+		if (locate(/obj/structure/lattice, T) || locate(/obj/structure/ladder, T) && !CanAvoidGravity() && !species.natural_climbing && (prob(8 * species.climb_coeff)))
```

Comment:
This probably also needs to check if there is gravity or not, otherwise they shouldn't be able to fall at all.

Reply:
Good catch, added